webpack.config.js

```diff
@@ -432,6 +432,25 @@ module.exports = {
                     "less-loader"
                 ]
             },
+            {
+                test: /\.scss$/,
+                use: [
+                    miniCssExtractPlugin.loader,
+                    "css-loader",
+                    "less-loader",
```

Comment:
Why do we need the less-loader for scss? I thought less and scss were independent from each other, and both "compile" to plain CSS?

Reply:
We don't - that was already removed before this review - I probably had it there by mistake.